update_deps/propagate_updates.ps1

```diff
@@ -98,6 +98,8 @@ function update-local-repo {
     cd $repo_name
     git checkout master
     git pull
+    # Remove deps folder to refresh
+    Remove-Item deps -Recurse -Force
```

Comment:
I don't understand why this is needed.

But if e are in paranoid state, why not run a git clean -xdff too while we are at it.

Reply:
Even I don't understand why dependency updates aren't being updated. This fixes it somehow.